src/lib/hooks/useFilteredFields.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import { EProjectFields } from '../consts/projects';
+
+export const useFilteredFields = (
+  watchedFields: any,
```

Comment:
do not use any as a type... in edge cases we use `as any` for type assertion when we know what it the required type and for some reason typescript didn't recognized this...

Reply:
I put the type ts suggested on ProjectsTable

Reply:
see comment https://github.com/shlomogcap/b-cont-react/pull/14#discussion_r1252624998